gopkg/da-rpc/near_test.go

```diff
@@ -1,7 +1,6 @@
 package near_test
 
 import (
-	"errors"
```

Comment:
Looks like the test is failing because of this

Reply:
Ah, I see, done

Reply:
Looks like all checks passed